bedrock/firefox/templates/firefox/whatsnew/includes/fx83/pocket.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="127" height="115" viewBox="0 0 127 115" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We're using THIS Pocket file because the Pocket SVG in Protocol doesn't have the white background for the downward-facing caret. 
<img width="114" alt="image" src="https://user-images.githubusercontent.com/2692333/97242863-f49dab00-17c2-11eb-98c1-14d5a07930b4.png">

Reply:
👍